MAUI/Kanban-Board/Column.md

```diff
@@ -78,7 +78,7 @@ More than one category can be mapped to a column by assigning multiple values to
 
 {% highlight C# %} 
 
-    progressColumn.Categories = new List<object>() { "In Progress", "Validated" };
```

Comment:
@EzhilarasanElangovan31 xaml code missing, update it within column, Categories="In Progress,Validated"

Reply:
Added xaml code